webpack/assets/javascripts/react_app/common/testHelpers.js

```diff
@@ -56,3 +56,14 @@ export const testComponentSnapshotsWithFixtures = (Component, fixtures) =>
   shallowRenderComponentWithFixtures(Component, fixtures)
     .forEach(({ description, component }) =>
       it(description, () => expect(toJson(component)).toMatchSnapshot()));
+
+/**
+ * Test a reducer with fixtures and snapshots
+ * @param  {Function} reducer  reduce function to test
+ * @param  {Object}   fixtures key=fixture description, value=props to apply
+ */
+export const testReducerSnapshotWithFixtures = (reducer, fixtures) => {
```

Comment:
Awesome :+1:

Reply:
Can you actually separate it (and the consuming of that at `BreadcrumbBarReducer.test.js`) into a different pr?

I want to start using it and I think we can merge it really quickly.

Reply:
Yeah sure, I will work on it today.

Reply:
@sharvit opened https://github.com/theforeman/foreman/pull/5516 to do as you asked